### PR TITLE
Add gaffer node metadata, fix bump space scaling

### PR DESCRIPTION
**Pull request by luisbarrancos**
The asGlass shader can use the Student T MDF as well, so specular spread was added.

### CODE REVIEW

File: src/appleseed.shaders/include/appleseed/maya/as_maya_helpers.h [hunk 1]
```diff
@@ -44,43 +44,49 @@
         string as_maya_attribute_name = "defaultColor",                     \
         int as_maya_attribute_hidden = 1,                                   \
         string label = "Default Color",                                     \
-        string page = "Color Balance"                                       \
+        string page = "Color Balance",                                      \
+        int gafferNoduleLayoutVisible = 0                                   \
```
Comment (dictoon) [inline src/appleseed.shaders/include/appleseed/maya/as_maya_helpers.h]: You did mean *Nodule* and not *Module* here right? Just checking :)

Comment (est77) [reply src/appleseed.shaders/include/appleseed/maya/as_maya_helpers.h]: Yes, it's Nodule

Comment (luisbarrancos) [reply src/appleseed.shaders/include/appleseed/maya/as_maya_helpers.h]: @dictoon yeah i know Franz... i don't like the mental image (no pun intended)[ it brings](https://www.merriam-webster.com/dictionary/nodule) either

Comment (Narann) [reply src/appleseed.shaders/include/appleseed/maya/as_maya_helpers.h]: Please, don't be me, don't type this on Google Image...

**Pull request merged**